engines/director/detection_tables.h

```diff
@@ -5266,40 +5470,30 @@ static const DirectorGameDescription gameDescriptions[] = {
-								  "PROGRAM/458MAIN.DXR",  "78244a687f279eda8655ac9fd7ce5531", 531620, 501),
-
-	MACGAME2("explorespace4", "", "Galaxy",			      "231d3041df162be4a0ddd36a74eb03b6", 719005,
-								  "PROGRAM/460MAIN.Dxr",  "4212a73dc3482104192ad4f0937446cf", 275180, 501),
-	WINGAME2("explorespace4",     "Win 95", "WIN95.EXE",  "1a7acbba10a7246ba58c1d53fc7203f5", 1411387,
-								  "PROGRAM/460MAIN.DXR",  "4212a73dc3482104192ad4f0937446cf", 275180, 501),
-	WINGAME2("explorespace4",     "Win 3.1", "WIN31.EXE", "842014a812b454620f64b860fd7211f0", 933175,
-								  "PROGRAM/460MAIN.DXR",  "4212a73dc3482104192ad4f0937446cf", 275180, 501),
-
-	MACGAME2("explorespace5", "", "Space",				  "231d3041df162be4a0ddd36a74eb03b6", 719005,
-								  "PROGRAM/459MAIN.Dxr",  "3868e98be38c2fbff814e2b8f1663fc9", 376868, 501),
-	WINGAME2("explorespace5", 	  "Win 95", "WIN95.EXE",  "1a7acbba10a7246ba58c1d53fc7203f5", 1411387,
-								  "PROGRAM/459MAIN.DXR",  "3868e98be38c2fbff814e2b8f1663fc9", 376868, 501),
-	WINGAME2("explorespace5",     "Win 3.1", "WIN31.EXE", "842014a812b454620f64b860fd7211f0", 933175,
-								  "PROGRAM/459MAIN.DXR",  "3868e98be38c2fbff814e2b8f1663fc9", 376868, 501),
+	MACGAME2("explorespace1", "", "Solar",				 "231d3041df162be4a0ddd36a74eb03b6", 719005,
+								  "PROGRAM/456MAIN.Dxr", "c6ce20cd8a95fd88aee9fabe8987aee6", 375330, 501),
+	WINGAME2("explorespace1", "", "WIN95.EXE",			 "1a7acbba10a7246ba58c1d53fc7203f5", 1411387,
+								  "PROGRAM/456MAIN.DXR", "c6ce20cd8a95fd88aee9fabe8987aee6", 375330, 501),
+
+	MACGAME2("explorespace2", "", "Mars",				 "231d3041df162be4a0ddd36a74eb03b6", 719005,
+								  "PROGRAM/457MAIN.Dxr", "9bb4f9407e4658f50ae970fc0a5133a6", 435968, 501),
+	WINGAME2("explorespace2", "", "WIN95.EXE",			 "1a7acbba10a7246ba58c1d53fc7203f5", 1411387,
+								  "PROGRAM/457MAIN.DXR", "9bb4f9407e4658f50ae970fc0a5133a6", 435968, 501),
+
+	MACGAME2("explorespace3", "", "Sun_Moon",			 "231d3041df162be4a0ddd36a74eb03b6", 719005,
+								  "PROGRAM/458MAIN.Dxr", "78244a687f279eda8655ac9fd7ce5531", 531620, 501),
+	WINGAME2("explorespace3", "", "WIN95.EXE",			 "1a7acbba10a7246ba58c1d53fc7203f5", 1411387,
+								  "PROGRAM/458MAIN.DXR", "78244a687f279eda8655ac9fd7ce5531", 531620, 501),
+
+	MACGAME2("explorespace4", "", "Galaxy",				 "231d3041df162be4a0ddd36a74eb03b6", 719005,
+								  "PROGRAM/460MAIN.Dxr", "4212a73dc3482104192ad4f0937446cf", 275180, 501),
+	WINGAME2("explorespace4", "", "WIN95.EXE",			 "1a7acbba10a7246ba58c1d53fc7203f5", 1411387,
+								  "PROGRAM/460MAIN.DXR", "4212a73dc3482104192ad4f0937446cf", 275180, 501),
+
+	MACGAME2("explorespace5", "", "Space",				 "231d3041df162be4a0ddd36a74eb03b6", 719005,
+								  "PROGRAM/459MAIN.Dxr", "3868e98be38c2fbff814e2b8f1663fc9", 376868, 501),
+	WINGAME2("explorespace5", "", "WIN95.EXE",			 "1a7acbba10a7246ba58c1d53fc7203f5", 1411387,
+								  "PROGRAM/459MAIN.DXR", "3868e98be38c2fbff814e2b8f1663fc9", 376868, 501),
 
```

Comment:
win 3,1 entries are unneeded?

Reply:
Yes, the general approach is to use, where available, Win9x binaries over Win3.x and PowerPC/fat binaries over 68k